README.md

```diff
@@ -241,6 +242,18 @@ Explanations:
 > git pull
 > ```
 
+
+### Fork Candy-doc
+
+The .github/workflows/publish-release.yml file uses these secrets :
+- GPG_PRIVATE_KEY
+- GPG_PASSPHRASE
+- MVN_CENTRAL_LOGIN
+- MVN_CENTRAL_PWD
+- GPG_KEY_NAME
+
+When you fork this project, add these secrets in your forked repository settings in order to publish your own version of Candy-doc on Maven Central.
```

Comment:
I don't think we want people to publish release of Candy-Doc on Maven Central. A message about "you will not able to publish release in a fork" is a better option in my opinion

Reply:
Done @ecattez